httpclient/metrics_round_tripper_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright © 2024 Acronis International GmbH.
+
+Released under MIT license.
+*/
+
+package httpclient
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewMetricsRoundTripper(t *testing.T) {
```

Comment:
The test currently doesn’t verify that all metric labels are set properly. For testing Prometheus metrics functionality, you can use the `testutil` package from go-appkit. Specifically, the `testutil.RequireSamplesCountInHistogram` function (https://github.com/acronis/go-appkit/blob/master/testutil/metrics.go#L35) could be helpful in your case.